scipy/stats/_stats_py.py

```diff
@@ -10244,9 +10463,17 @@ def wasserstein_distance(u_values, v_values, u_weights=None, v_weights=None):
     References
     ----------
     .. [1] "Wasserstein metric", https://en.wikipedia.org/wiki/Wasserstein_metric
-    .. [2] Ramdas, Garcia, Cuturi "On Wasserstein Two Sample Testing and Related
+    .. [2] Lili Weng, "What is Wasserstein distance?", Lil'log,
+           https://lilianweng.github.io/posts/2017-08-20-gan/#what-is-
+           wasserstein-distance.
```

Comment:
Link is not rendering correctly, so I suppose it needs to be all on one line. Feel free to make it `"""  # noqa: e501` at the end of the docstring if the line is too long.

Reply:
Linter didn't give an error for me locally. Let's see how it goes here.